truss/util/path.py

```diff
@@ -1,33 +1,57 @@
+import fnmatch
 import os
 import random
 import string
 import tempfile
 from contextlib import contextmanager
-from distutils.dir_util import copy_tree, remove_tree
 from distutils.file_util import copy_file
 from pathlib import Path
-from typing import List, Optional, Tuple, Union
+from shutil import copytree
+from typing import List, Optional, Tuple
 
 from truss.patch.hash import str_hash_str
 
+TRUSS_IGNORE_PATH = Path(__file__).parent / ".truss_ignore"
 
-def copy_tree_path(src: Path, dest: Path) -> List[str]:
-    return copy_tree(str(src), str(dest), verbose=0)
+
+def copy_tree_path(
+    src: Path, dest: Path, skip_directories: Optional[List[str]] = None
```

Comment:
skip_directories sounds like a verb, may be rename to `directories_to_skip` here and other places.